src/code/z_actor.c

```diff
@@ -265,7 +265,7 @@ typedef struct AttentionColor {
     /* 0x04 */ Color_RGBA8 secondary; // Used for Navi's outer color
 } AttentionColor;                     // size = 0x8
 
-AttentionColor sAttentionColors[ACTORCAT_MAX + 1] = {
+static AttentionColor sAttentionColors[ACTORCAT_MAX + 1] = {
```

Comment:
can you comment on why this needs to be static (at the very least "prevents EGCS crash" or "needed for EGCS matching")